src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -67,19 +69,25 @@ public TPCCLoader(TPCCBenchmark benchmark) {
         }
     }
 
+    private boolean deferConstraintChecks;
     private int numWarehouses = 0;
     private static final int FIRST_UNPROCESSED_O_ID = 2101;
 
     @Override
     public List<LoaderThread> createLoaderThreads() throws SQLException {
         List<LoaderThread> threads = new ArrayList<LoaderThread>();
         final CountDownLatch itemLatch = new CountDownLatch(1);
+        final CountDownLatch warehouseLatch = new CountDownLatch(numWarehouses);
 
         // ITEM
         // This will be invoked first and executed in a single thread.
         threads.add(new LoaderThread() {
             @Override
             public void load(Connection conn) throws SQLException {
+                if (!workConf.getDeferConstraintChecks()) {
+                    EnableForeignKeyConstraints(conn);
+                    LOG.info("The total Time for enabling foreign keys " + (System.nanoTime() - startTime) / 1000 / 1000 / 1000);
```

Comment:
We can skip this log since this is not a time consuming operation.

Reply:
done.